examples/quickstart/jupyter-notebooks/notebooks/04-api/01-delete-api-tutorial.ipynb

```diff
@@ -0,0 +1,975 @@
+    "2015-09-12T14:00:00.000Z_2015-09-12T15:00:00.000Z\n",
+    "2015-09-12T15:00:00.000Z_2015-09-12T16:00:00.000Z\n",
+    "2015-09-12T16:00:00.000Z_2015-09-12T17:00:00.000Z\n",
+    "2015-09-12T17:00:00.000Z_2015-09-12T18:00:00.000Z\n",
+    "2015-09-12T20:00:00.000Z_2015-09-12T21:00:00.000Z\n",
+    "2015-09-12T21:00:00.000Z_2015-09-12T22:00:00.000Z\n",
+    "2015-09-12T22:00:00.000Z_2015-09-12T23:00:00.000Z\n",
+    "2015-09-12T23:00:00.000Z_2015-09-13T00:00:00.000Z\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0d0578c",
+   "metadata": {},
+   "source": [
+    "## Delete entire table"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b8d0260a",
+   "metadata": {},
+   "source": [
+    "You can delete entire tables the same way you can delete parts of a table, using intervals.\n",
+    "\n",
+    "Run the following cell to reset the endpoint to `/druid/coordinator/v1/datasources/:dataSource/markUnused`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd354886",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = druid_host + '/druid/coordinator/v1/datasources/wikipedia_hour/markUnused'\n",
+    "endpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed3cf7d3",
+   "metadata": {},
+   "source": [
+    "Next, send a `POST` with the payload `{\"interval\": \"2015-09-12T00:00:00.000Z/2015-09-13T01:00:00.000Z\"}` to mark the entirety of the table as \"unused.\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25639752",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql_request = {\n",
+    "  \"interval\": \"2015-09-12T00:00:00.000Z/2015-09-13T01:00:00.000Z\"\n",
+    "}\n",
+    "\n",
+    "response = session.post(endpoint, json=sql_request);\n",
+    "\n",
+    "response.status_code"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bbbba823",
+   "metadata": {},
+   "source": [
+    "To verify the segment changes, the following cell sets the endpoint to `/druid/v2/sql` and send a SQL-based request. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eac1db4c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = druid_host + '/druid/v2/sql'\n",
+    "sql_request = {\n",
+    "  \"query\": \"SELECT segment_id FROM sys.segments WHERE \\\"datasource\\\" = 'wikipedia_hour'\",\n",
+    "  \"resultFormat\": \"objectLines\"\n",
+    "}\n",
+    "\n",
+    "response = session.post(endpoint, json=sql_request);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "deae8727",
+   "metadata": {},
+   "source": [
+    "Run the next cells to view the response. You should see that the `response.text` returns nothing, but `response.status_code` returns a 200. \n",
+    "\n",
+    "The response should return the remaining segments, but since the table was deleted, there are no segments to return."
```

Comment:
Active voice 
```suggestion
    "The response returns the remaining segments, but since you deleted the table, there are no segments to return."
```